libraries/dotnet/EnergyOrigin.ActivityLog/EnergyOrigin.ActivityLog/API/ActivityLogEntryResponse.cs

```diff
@@ -22,5 +22,5 @@ public enum ActionTypeEnum { Created, Accepted, Declined, Activated, Deactivated
     // Action
     public EntityTypeEnum EntityType { get; init; }
     public ActionTypeEnum ActionType { get; init; }
-    public Guid EntityId { get; init; }
+    public string EntityId { get; init; } = "";
```

Comment:
Why should it be a string?

Reply:
Because we want to display GSRN number and not contract ID fx